Box.V2/Models/BoxApplication.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Box.V2.Models
+{
+    /// <summary>
+    /// Box representation of an application.
+    /// </summary>
+    class BoxApplication : BoxEntity
+    {
+        public const string FieldName = "name";
+        public const string FieldApiKey = "api_key";
+
+        /// <summary>
+        /// The name of this application.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldName)]
+        public String Name { get; set; }
```

Comment:
Should the setters for these properties be private like the other event source models in this patchset?